src/shaders/_prelude.vertex.glsl

```diff
@@ -35,17 +35,25 @@ vec4 evaluate_zoom_function_4(const vec4 value0, const vec4 value1, const vec4 v
     }
 }
 
+// Unpack a pair of values that have been packed into a single float.
+// The packed values are assumed to be 8-bit unsigned integers, and are
+// packed like so:
+// packedValue = floor(input[0]) * 256 + input[1],
+vec2 unpack_float(const float packedValue) {
+    float v0 = floor(packedValue / 256.0);
```

Comment:
It's specified in [OpenGL® ES Shading Language](https://www.khronos.org/files/opengles_shading_language.pdf) §4.5.2 (p33). For `highp`, 16 bits of precision, integer range (-2¹⁶, 2¹⁶).